docs/navigation.html

```diff
@@ -0,0 +1,460 @@
+                            <polygon points="-2.16506,-2.5 2.16506,0 -2.16506,2.5" fill="black" transform="scale(0) translate(13.125 10)" style="transform-origin: 13.125px 10px;">
+                            </polygon>
+                            <circle fill="black" cx="12.5" cy="10" r="0"></circle>
+                        </mask>
+                        <rect fill="hsl(139, calc(var(--saturation-factor, 1) * 47.3%), 43.9%)" width="25" height="15" mask="url(#2b3fa5a7-0ec2-4469-b164-f8d5007235b6)"></rect>
+                    </svg>
+                    <rect x="22" y="22" width="10" height="10" fill="transparent" aria-hidden="true" class="pointerEvents-2zdfdO"></rect>
+                </svg>
+                <div class="flex flex-column">
+                    <span class="font-white">Made 😎 by Mehul </span>
+                    
+                </div>
+             </div>
+        </aside>
+    </nav>
+
+    <div class="container">
+        <div class="mt-3">
+            <h1 class="pb-1">Navbars</h1>
+            <p class="pb-1">To make navbar stick to the top add <span class="text-red">.navbar-fixed</span> class</p>
+            <p class="pb-1">While changing the color of navbar make sure to change color of text to match navbar color. we have 
+            <span class="text-red">.navbar-danger .navbar-primary </span> colors for navbars.</p>
+            <p class="pb-1"> make sure when you add colors to nav add <span class="text-red">.navbar-white-text</span>class to the texts.</p>
+            <div class="card mt-1">
+
+                <div class="pt-2 pb-2">
+
+                    <nav class="navbar flex flex-wrap  flex-between">
+                        <div class="flex flex-space-evenly flex-align-item-center ">
+                            <ul class=" text-deocration-none flex flex-wrap flex-align-item-center gap">
+                                <li class="pb-1 nav-text navbar-dark-text flex flex-align-item-center gap-1">
+                                    <span class="material-icons snackbar-icons py-1 navbar-icons">menu</span>
+                                </li>
+                                <a href="#" class="text-deocration-none">
+                                    <li class="pb-1 nav-text navbar-dark-text flex flex-align-item-center gap-1">
+                                        <span class="font-bold">Zeplin</span>
+                                    </li>
+                                </a>
+                            </ul>
+                        </div>
+
+                        <div class="flex flex-space-evenly flex-align-item-center ">
+                            <ul class=" text-deocration-none flex flex-wrap gap-col-2">
+                                <a href="#" class="text-deocration-none" target="_blank" rel="noreferrer">
+                                    <li class="pb-1 nav-text navbar-dark-text flex flex-align-item-center gap-1">
```

Comment:
fix: These links would not be target=blank beacause in navbar we want to navigate within our app

Reply:
sure will make the changes

Reply:
> sure will make the changes



> fix: These links would not be target=blank beacause in navbar we want to navigate within our app

well @Ankur9669 its upto the user if they want to remove it they can remove it when they  implement in their app